cvat/apps/engine/lazy_list.py

```diff
@@ -180,6 +180,20 @@ def _parse_up_to(self, index: int) -> None:
         if self._parsed:
             return
 
+        if index == -1:
+            # _iter_unparsed is not effective when we want to parse the whole list
+            splitted = self._string.split(self._separator)
```

Comment:
```suggestion
            element_strs = self._string.split(self._separator)
```

"Splitted" is ungrammatical.